prover/drivers/kore-driver.md

```diff
@@ -57,6 +70,15 @@ Add various standard Kore declarations to the configuration directly:
        </declarations>
 ```
 
+For hook-smt-symbol, hook-smt-sort
+```k
+  rule <k> hook-smt-symbol(From, To) => .K ...</k>
+       <hooked-smt-symbols>... (.Map => (From |-> To)) </hooked-smt-symbols>
```

Comment:
Lets use axioms here instead of introducing new cells.

Reply:
As above.